scripts/update/transmission.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [ -f /install/.transmission.lock ]; then
+
+    if ! grep -q /home/systemd/system/transmission@.service "--logfile"; then
+        echo_progress_start "Moving transmission's logs to a file instead of syslog"
+        : #switch service to logfile
+        sed "s/ExecStart=.*/ExecStart=/usr/bin/transmission-daemon -f --log-error --logfile /home/%i/.config/transmission-daemon/transmission.log/g" -i
```

Comment:
This is not the same command as the install script and also the forward slashes are going to cause this command to fail.

Reply:
I don't see the difference, can you point it out please?

Reply:
Oh I may have conflated the `-i` on the end for sed as part of the `ExecStart` command